### DISCUSSION TITLE
Fanmade Extension: Gravity (Updated)

### DISCUSSION THREAD

**Discussion by mcnole25** [TurboWarp/extensions#1246] [Show and tell]
I made another extension called Gravity, to make platforming and particle games easier to code. It also supports clones having their own velocity.

![Screenshot 2024-01-18 at 02-01-12 TurboWarp - Run Scratch projects faster](https://github.com/TurboWarp/extensions/assets/154467369/f60c53cb-0838-4a3f-93c0-9d59e8426138)
[gravity.txt](https://github.com/TurboWarp/extensions/files/13973138/gravity.txt)

Note that it obviously must be run unsandboxed.
Also, let me know if there's any bugs.

**Comment by LilyMakesThings** [comment]
I read this as "another family guy extension"

**Reply by mcnole25** [reply]
Note: I couldn't add "collision with sprite" blocks, cause I'm not a skilled JS coder. You have to code it in yourself.
The reason is that it could be hard to code. It's an extension I want to make eventually, but not yet.

**Comment by unknown07724** [comment]
doesn't box2d physics already do this? if yes than probably make this into a newtonian gravity extension

**Comment by LegoBrainBiker** [comment]
this doesn't seem very uyseful, it could be implemented with blocks easily. also, consider adding a step (deltatime) block, that could be useful
